cmd/assign/address/address.go

```diff
@@ -43,7 +43,7 @@ func CmdAssignAddress(c *cli.Context) {
 
 	for _, teamDrive := range teamDrives {
 		if teamDrive.Name == c.Args().Get(0) {
-			_, err := driveApi.CreatePermission(teamDrive.Id, "organizer", c.Args().Get(1), "user")
+			_, err := driveApi.CreatePermission(teamDrive.Id, "organizer", c.Args().Get(1), c.Args().Get(2))
```

Comment:
Can you add a check if the flag is provided and print an error if not and return?